validator/lib/validateBoundingVolume.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var Promise = require('bluebird');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var Rectangle = Cesium.Rectangle;
+var Cartographic = Cesium.Cartographic;
+
+module.exports = validateVolume;
+
+function validateVolume(tileset) {
+    return new Promise(function(resolve) {
+        validateNode(tileset.root, tileset, resolve);
+    });
+}
+
+var scratchCartographic = new Cartographic();
+
+function regionInsideRegion(contentRect, tileRect) {
+    return (Rectangle.contains(tileRect, Cartographic.fromRadians(contentRect.west,
+        contentRect.north, 0.0, scratchCartographic))
+    && Rectangle.contains(tileRect, Cartographic.fromRadians(contentRect.west,
+        contentRect.south, 0.0, scratchCartographic))
+    && Rectangle.contains(tileRect, Cartographic.fromRadians(contentRect.east,
+        contentRect.north, 0.0, scratchCartographic))
+    && Rectangle.contains(tileRect, Cartographic.fromRadians(contentRect.east,
+        contentRect.south, 0.0, scratchCartographic)));
+}
+
+function sphereInsideSphere(contentSphere, tileSphere) {
```

Comment:
Yeah that's the end goal. Our plan was to write them here first and then push them over. Ideally we should get these into core Cesium before 12/1.